test/Microsoft.Extensions.Configuration.UserSecrets.Test/ConfigurationExtensionTest.cs

```diff
@@ -43,6 +45,35 @@ private void SetSecret(string id, string key, string value)
         }
 
         [Fact]
+        public void AddUserSecrets_FindsAssemblyAttribute()
+        {
+            var configKey = nameof(AddUserSecrets_FindsAssemblyAttribute);
+            SetSecret("Microsoft.Extensions.Configuration.UserSecrets.Test", configKey, "true");
```

Comment:
The secret id and values aren't important to the test. I can make them dummy values if it makes the purpose of the test clearer.